verilog/preprocessor/verilog_preprocess.cc

```diff
@@ -199,6 +201,24 @@ std::unique_ptr<VerilogPreprocessError> VerilogPreprocess::ParseMacroDefinition(
   return nullptr;
 }
 
+// Responds to `define directives.  Macro definitions are parsed and saved
+// for use within the same file.
+absl::Status VerilogPreprocess::HandleMacroIdentifier(
+    const TokenStreamView::const_iterator iter  // points to `MACROIDENTIFIER token
+    ) {
+    const absl::string_view sv = (*iter)->text();
```

Comment:
At some point, you also want to make this dependent on the config (that now only has the `filter_branches` field). There might be situations in which we do not want to expand macros, so this will help with that (and i suspect a bunch of tests fail now because they assume that macro expansion does not take place, so there you then can set the values you need for the particular test; so maybe you need to support such configuration option rather earlier than later)